test/SourceKit/Diagnostics/rdar107281079.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %sourcekitd-test -req=diags %s -- %s -I %S/../Inputs/header_with_macro | %FileCheck %s
```

Comment:
Test name is fairly opaque to OSS 😅. `cross_language_diagnostic.swift` or something?